content/en/docs/concepts/instrumentation/assisted.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Assisted
+description: >-
+  Learn about ways to simplify the process of instrumenting your code.
+weight: 30
+---
+
+Different to
+[Automatic Instrumentation](/docs/concepts/instrumentation/automatic/) an
+**Assisted Instrumentation** requires configuration or code changes, yet they
+provide a convenient way to quickly add OpenTelemetry to your application.
+
+An example for a technology that assists instrumentation are
+[**Instrumentation Libraries**](/docs/concepts/instrumentation/libaries), which
+are added by a developer to their application to extract telemetry from their
+dependencies. By using them, they are no longer required to write a wrapper
+themselves. Although they are an intermediate solution, until the libraries
+provide out of the box support for OpenTelemetry.
```

Comment:
I'm not sure I got the meaning of the last sentence correct, so please adjust as needed!

```suggestion
An example of a technology that assists instrumentation is an
[**Instrumentation Library**](/docs/concepts/instrumentation/libraries), which
is added by a developer to their application to extract telemetry from their
dependencies. By using libraries, developers are no longer required to write a wrapper
themselves. Instrumentation libraries are an intermediate solution until the libraries
provide out-of-the-box support for OpenTelemetry.
```